kafka/README.md

```diff
@@ -1 +1,55 @@
 # Adaptive Alerting - Kafka
+
+
+## Running KStream apps in docker
+
+Need to package the app (if other modules have been updated those will need to be re-built):
+
+```bash
+mvn clean package
+```
+
+Build the docker image:
+
+```bash
+docker build -f docker/{APP_NAME e.g. constant-detector}/Dockerfile .
```

Comment:
How about tagging build with app name, so that we don't se anonymous images ? 

`docker build -t {APP_NAME e.g. constant-detector}  -f docker/{APP_NAME}/Dockerfile . `
`docker run {APP_NAME}`